website/static/js/profile.js

```diff
@@ -577,6 +609,41 @@ var ListViewModel = function(ContentModel, urls, modes) {
         }
         return true;
     });
+    
+
+    self.institutionsEmpty = ko.computed(function() {
+
+            for (var i=0; i<self.contents().length; i++) {
+                if (self.contents()[i].institutionEmpty()) { 
+                    return true;
+                }
+            }
+ 
+        return false;
+                
+    });
+
+    
+    self.extraFieldsEmpty = ko.computed(function() {
+        var i;
```

Comment:
I'd really prefer that counter variables be defined in the `for` declaration.

It doesn't make a difference from a logical perspective, but it will in the future with ES6. It's also just plain old convention, and my first reaction when seeing `for(i=0` is to think that the author mistakenly created a global variable by omitting the `var` keyword.